components/ReplyFeedback.js

```diff
@@ -34,24 +181,24 @@ const ArticleReplyFeedbackForUser = gql`
 const ArticleReplyFeedbackData = gql`
   fragment ArticleReplyFeedbackData on ArticleReply {
     ...ArticleReplyFeedbackForUser
-    user {
-      id
-    }
     positiveFeedbackCount
     negativeFeedbackCount
+    ownVote
     feedbacks {
```

Comment:
You also need `vote` under each feedbacks, because the you are using `vote` to move feedbacks in the right tab.

Currently both tabs are empty because no feedback has `vote` field......
![image](https://user-images.githubusercontent.com/108608/80891094-61445b00-8cf4-11ea-82a2-eda5e2b83054.png)